### DISCUSSION TITLE
Tailwind utility for bg-stripes not in the docs?

### DISCUSSION THREAD

**Discussion by las3r** [tailwindlabs/tailwindcss#3229] [Help] [answered]
Hi there,

In the tailwind docs I see the following utility classes being used:
`bg-stripes` and `bg-stripes-white`.

Example:
https://tailwindcss.com/docs/box-sizing -> See the striped cubes in the page.

This utility does not seem available in (at least) my bundle. Is this a hidden gem that we need to request ;-)?

**Comment by adamwathan** [comment, marked as answer]
Just a custom class we created for the docs! Tailwind is super easy to extend with any of your own custom classes like this whenever needed 👍🏻

**Reply by las3r** [reply]
I understand and I love it. here's hoping for it to be included in as a helper.

**Comment by mpalpha** [comment]
@las3r https://play.tailwindcss.com/QNkCU1z1UX?file=config
